lib/css/modules/_orangered.scss

```diff
@@ -25,6 +25,13 @@
 	}
 }
 
+.res-orangered-hideEmptyModmailButton {
```

Comment:
Hopefully the update works. I'm a beginner so a lot of this is actually flying over my head; it's not clear to me what you're asking for in request 1. I may have bitten off more than I can chew. Sorry about that.